airbyte-integrations/connectors/source-aircall/source_aircall/manifest.yaml

```diff
@@ -17,19 +17,18 @@ definitions:
       extractor:
         type: DpathExtractor
         field_path: ["{{ parameters.name }}"]
-      paginator:
-        type: "DefaultPaginator"
-        page_size_option:
-          type: "RequestOption"
-          inject_into: "request_parameter"
-          field_name: "{{ response['meta']['per_page'] }}"
-        pagination_strategy:
-          type: "PageIncrement"
-          page_size: 5
-        page_token_option:
-          type: "RequestOption"
-          inject_into: "request_parameter"
-          field_name: "{{ response['meta']['page'] }}"
+    paginator:
+      type: "DefaultPaginator"
+      page_size_option:
+        type: "RequestOption"
+        inject_into: "request_parameter"
+        field_name: "{{ response['meta']['per_page'] }}"
```

Comment:
This still looks sus — shouldn't field_name be just `per_page` or something?

Reply:
Oh shoot, this is the same thing I fixed for `page_token_option` haha. Yeah that's right, I'll fix.

Reply:
fixed! ty